src/components/TableComponents/TicketsBodyComponents/TicketsBodyComponents.tsx

```diff
@@ -16,41 +17,44 @@ const TicketsBodyComponents = ({
   page,
   rowsPerPage,
 }: ITicketsBodyComponents) => {
-  const tableData = TICKETS_DATA.slice(
+  const { data } = useAppSelector((store) => store?.getTickets);
```

Comment:
You don't need an optional chaining here, as store will be always initialised. (default state at least)